Frosting project fails on Linux

### What You Are Seeing?
When trying to run a Frosting project created following the [Getting started guide](https://cakebuild.net/docs/getting-started/setting-up-a-new-frosting-project) an exception is thrown when running on Linux.

### What is Expected?
Projects created using the default template / documentation should work on all platforms.

### What version of Cake are you using?
1.0.0-rc0002

### Are you running on a 32 or 64 bit system?
64-Bit

### What environment are you running on?  Windows? Linux? Mac?
Linux

### Are you running on a CI Server?  If so, which one?
Azure Pipelines

https://dev.azure.com/pberger/Cake-Test/_build/results?buildId=135&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=33e5d3ec-87e7-5f80-0281-074c6962cb44&l=35

### How Did You Get This To Happen? (Steps to Reproduce)

1. Create a new Frosting project based on default template and following documentation
2. Run on Azure Pipelines (Ubunto 18.04 Image; Only one bash task calling to `build.sh`)

Repro repo: https://github.com/pascalberger/cake-frosting-example

### Output Log

~~~sh
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.179.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
Formatted command: exec bash '/home/vsts/work/1/s/build.sh'
========================== Starting Command Output ===========================
/bin/bash --noprofile --norc /home/vsts/work/_temp/db5af56e-9d60-4d08-a79a-222214018edb.sh

Welcome to .NET 5.0!
---------------------
SDK Version: 5.0.101

Telemetry
---------
The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.

Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry

----------------
Installed an ASP.NET Core HTTPS development certificate.
To trust the certificate run 'dotnet dev-certs https --trust' (Windows and macOS only).
Learn about HTTPS: https://aka.ms/dotnet-https
----------------
Write your first app: https://aka.ms/dotnet-hello-world
Find out what's new: https://aka.ms/dotnet-whats-new
Explore documentation: https://aka.ms/dotnet-docs
Report issues and find source on GitHub: https://github.com/dotnet/core
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------
System.ComponentModel.Win32Exception (2): No such file or directory
   at System.Diagnostics.Process.ForkAndExecProcess(String filename, String[] argv, String[] envp, String cwd, Boolean redirectStdin, Boolean redirectStdout, Boolean redirectStderr, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at Microsoft.DotNet.Cli.Utils.Command.Execute(Action`1 processStarted)
   at Microsoft.DotNet.Cli.Utils.Command.Execute()
   at Microsoft.DotNet.Tools.Run.RunCommand.Execute()
   at Microsoft.DotNet.Tools.Run.RunCommand.Run(String[] args)
   at Microsoft.DotNet.Cli.Program.ProcessArgs(String[] args, ITelemetry telemetryClient)
   at Microsoft.DotNet.Cli.Program.Main(String[] args)
##[error]Bash exited with code '1'.
Finishing: Bash Script
~~~

--- a/src/Cake.Frosting.Template/templates/cakefrosting/build/Build.csproj
+++ b/src/Cake.Frosting.Template/templates/cakefrosting/build/Build.csproj
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <RunWorkingDirectory>$(MSBuildProjectDirectory)..</RunWorkingDirectory>
+    <RunWorkingDirectory>$(MSBuildProjectDirectory)</RunWorkingDirectory>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Cake.Frosting" Version="1.0.0" />

--- a/src/Cake.Frosting.Template/templates/cakefrosting/build/Program.cs
+++ b/src/Cake.Frosting.Template/templates/cakefrosting/build/Program.cs
@@ -9,6 +9,7 @@ public static class Program
     {
         return new CakeHost()
             .UseContext<BuildContext>()
+            .UseWorkingDirectory("..")
             .Run(args);
     }
 }

